src/components/grid/GridComponent.vue

```diff
@@ -198,7 +198,7 @@ export default Vue.extend({
   },
   filters: {
     formatCount: function (value) {
-      if (value === -1) {
```

Comment:
server should never return values lower the cutoff point, so i do not understand this check

Reply:
the -1 is a molgenis thing, that says there are values, but they are below threshold.
So that is something you want to know.